From f29e8c4d92c50ec22b9f96455ed809d8020a2b06 Mon Sep 17 00:00:00 2001
From: VishalKumar <vkvishal060991@gmail.com>
Date: Sun, 19 May 2019 22:46:13 +0530
Subject: [PATCH]  commit for checking patch

---
 chkout.txt | 1 +
 1 file changed, 1 insertion(+)

diff --git a/chkout.txt b/chkout.txt
index 1314837..65581b6 100644
--- a/chkout.txt
+++ b/chkout.txt
@@ -1 +1,2 @@
 echo " file to test checkout command "
+echo " it is to check a patch "
-- 
2.17.1.windows.2

